utils/chain-ops/genesis/config.go

```diff
@@ -102,8 +102,6 @@ type DeployConfig struct {
 	SystemConfigProxy common.Address `json:"systemConfigProxy"`
 	// KromaPortal proxy address on L1
 	KromaPortalProxy common.Address `json:"kromaPortalProxy"`
-	// Colosseum proxy address on L1
-	ColosseumProxy common.Address `json:"colosseumProxy"`
```

Comment:
Do you have any reason of removing this `ColosseumProxy`?

Reply:
This is unnecessary. It is not used anywhere.